.travis.yml

```diff
@@ -31,37 +28,15 @@ script:
 
 jobs:
   include:
-    - php: 5.5
-      env: deps=low SYMFONY_DEPRECATIONS_HELPER=weak
-      install:
-        - travis_retry composer update -n --prefer-lowest --prefer-stable --prefer-dist
-
-    - php: 5.6
-      env: SYMFONY_VERSION="2.8.*"
-
-    - php: 7.0
-      env: SYMFONY_VERSION="3.2.*"
-
-    - php: 7.1
-      env: stability=RC SYMFONY_DEPRECATIONS_HELPER=weak SYMFONY_VERSION="3.4.*"
-      install:
-        - travis_retry composer update -n --prefer-dist
-
-    - php: 7.1
+    - php: 7.2
       env: stability=dev SYMFONY_DEPRECATIONS_HELPER=weak SYMFONY_VERSION="4.1.*"
       install:
         - composer require --dev "symfony/messenger" --no-update
         - travis_retry composer update -n --prefer-dist
 
-    - php: 7.2
-      env: stability=alpha
-      install:
-        - composer config minimum-stability alpha
-        - travis_retry composer update -n --prefer-dist
-
     - stage: Code Quality
       env: CODING_STANDARDS
-      php: 7.1
+      php: 7.2
       install:
         - travis_retry composer require -n --prefer-dist --dev doctrine/coding-standard:^5.0
```

Comment:
We can now move this to composer.json directly. 👍

Reply:
:+1: but would need the usual `composer.lock` ballet too

Reply:
Let's keep it to separate PR, we can enable some 7.1+ sniffs that were excluded and also use `use function`. 😉 cc @carusogabriel if you're interested